packages/shared/src/components/cards/v1/RaisedLabel.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { ReactElement } from 'react';
+import classNames from 'classnames';
+import { SimpleTooltip } from '../../tooltips/SimpleTooltip';
+
+export enum RaisedLabelType {
+  Hot = 'Hot',
+  Pinned = 'Pinned',
+  Beta = 'Beta',
+}
+
+const typeToClassName: Record<RaisedLabelType, string> = {
+  [RaisedLabelType.Hot]: 'bg-theme-status-error',
+  [RaisedLabelType.Pinned]: 'bg-theme-bg-bun',
+  [RaisedLabelType.Beta]: 'bg-theme-bg-cabbage',
+};
+
+export interface RaisedLabelProps {
+  type: RaisedLabelType;
+  description?: string;
+  className?: string | undefined;
+  focus?: boolean;
+}
+
+export function RaisedLabel({
+  type = RaisedLabelType.Hot,
+  description,
+  className,
+  focus = false,
+}: RaisedLabelProps): ReactElement {
+  return (
+    <div
+      className={classNames(
+        'absolute right-2 flex flex-row px-2 group-hover:bg-theme-bg-secondary group-focus:-top-0.5 group-focus:bg-theme-bg-secondary',
+        !focus && '-top-[1px] bg-theme-bg-primary',
+        focus && '-top-0.5 bg-theme-bg-secondary',
+        className,
+      )}
+    >
+      <SimpleTooltip content={description}>
+        <div
+          className={classNames(
+            'relative -top-2 h-4 rounded-sm px-2 font-bold uppercase text-white typo-caption2',
```

Comment:
Just to be inline with what we have.
```suggestion
            'relative -top-2 h-4 rounded-2 px-2 font-bold uppercase text-white typo-caption2',
```

Reply:
I think we probably can move `radius` to overwrite not extend to be honest, this will avoid this in the future.
I'll raise it in web-guild.

Reply:
should be 4px, so changing to `rounded`